models/action.go

```diff
@@ -334,6 +335,17 @@ func GetFeeds(opts GetFeedsOptions) ([]*Action, error) {
 		cond = cond.And(builder.Eq{"is_deleted": false})
 	}
 
+	if opts.Date != "" {
+		var dateLow time.Time
+		var dateHigh time.Time
+
+		dateLow, _ = time.Parse("2006-1-2", opts.Date)
```

Comment:
Shouldn't ignore the error.

Reply:
```suggestion
		var err error
		dateLow, err = time.Parse("2006-1-2", opts.Date)
		if err != nil {
			return fmt.Errorf("Unable to parse %s: %v", opts.Date, err)
		}
```

should do it.

Reply:
I think we have to ways to handle the error. The way @zeripath suggested or by just logging the error. Returning the error will cause a 500 error code. Logging the error seems to be closer to the behavior on other parts of gitea. For example if I use invalid parameters for sorting the issue list it just don't sort instead of throwing an error.

I'm not sure which way is better. What do you think?

```go
dateLow, err := time.Parse("2006-1-2", opts.Date)
if err != nil {
	log.Warn("Unable to parse %s, filter not applied: %v", opts.Date, err)
} else {
	dateHigh := dateLow.Add(86399000000000) // 23h59m59s

	cond = cond.And(builder.Gte{"created_unix": dateLow.Unix()})
	cond = cond.And(builder.Lte{"created_unix": dateHigh.Unix()})
}
```

Reply:
I like yours because if we fail to parse the date, the functions below would fail as well (panic?).